src/components/PullRequestHeading.tsx

```diff
@@ -0,0 +1,33 @@
+import { Avatar, Box, Heading, Text } from "@chakra-ui/react";
+import React from "react";
+
+import { ReviewPullRequestType } from "@/types/PullRequestType";
+
+type Props = {
+  owner: string | string[] | undefined;
+  repo: string | string[] | undefined;
+  pullRequest: ReviewPullRequestType;
+};
+
+const PullRequestHeading = ({ owner, repo, pullRequest }: Props) => {
+  return (
+    <Box width={700} align="start">
```

Comment:
```suggestion
    <>
```
これレビューページの「質問」「素敵」「改善」を固定するときに試行錯誤した名残な気がする、、、
ただ `align='start'` に関しては[ここ](https://github.com/git-baboo/easy-review/blob/develop/src/pages/%5Bowner%5D/%5Brepo%5D/%5BpullNumber%5D/review.tsx#L181)の打ち消しの意味があるようで、削除するとレビューページで右寄せなってしまう
けどその `align='end'` も同じく名残と思われる
(結論、一旦置いといて `align='end'` とともに別 Issue で消すか、どうせ消すから一旦表示が崩れることは分かった上で今消しちゃうか)

Reply:
一旦残して、別 Issue を立てておいて欲しいかもです！